akka-http-core/src/main/scala/akka/http/impl/engine/rendering/HttpResponseRendererFactory.scala

```diff
@@ -33,22 +33,23 @@ private[http] class HttpResponseRendererFactory(serverHeader: Option[headers.Ser
     }
 
   // as an optimization we cache the Date header of the last second here
-  @volatile private[this] var cachedDateHeader: (Long, Array[Byte]) = (0L, null)
-
+  private[this] var cachedDateHeader: (Long, Array[Byte]) = (0L, null)
   private def dateHeader: Array[Byte] = {
     var (cachedSeconds, cachedBytes) = cachedDateHeader
```

Comment:
I believe it does. It would probably be more efficient to do
```scala
var cachedSeconds = cachedDateHeader._1
var cachedBytes = cachedDateHeader._2
```
Unsure how much of a performance difference it'd make in this case though.